lib/core/re.nit

```diff
@@ -366,19 +379,19 @@ class Regex
 
 		# Found one?
 		if res == 0 then
-			var bso = bstr.byte_to_char_index(native_match.rm_so)
-			var ln = bstr.byte_to_char_index(native_match.rm_eo - native_match.rm_so - 1)
+			var first_char = bstr.byte_to_char_index(native_match.rm_so)
+			var last_char = bstr.byte_to_char_index(native_match.rm_eo - native_match.rm_so - 1)
```

Comment:
The more I look at this line, the wronger it seems.

I'll investigate later, but in the meantime I guess this can be merged